code/game/objects/items/weapons/twohanded.dm

```diff
@@ -432,3 +432,75 @@
 			M.faction = user.faction.Copy()
 			M.Copy_Parent(user, 100, user.health/2.5, 12, 30)
 			M.GiveTarget(L)
+
+/obj/item/weapon/twohanded/ram
+	name = "kinetic battering ram"
+	desc = "Internal gyroscopes give this ram excellent balance and the necessary force to break doors... or ribs."
+	icon_state = "ram0"
+	force = 10
+	force_wielded = 24
```

Comment:
best solution would probably just be to lower the force_wielded and have it apply a short stun as well.  oxy_loss might be kinda memey and nice as well.